cdap-api/pom.xml

```diff
@@ -29,6 +29,11 @@
   <artifactId>cdap-api</artifactId>
   <name>CDAP API</name>
   <packaging>jar</packaging>
+  <properties>
+    <!-- Workaround for bug http://jira.codehaus.org/browse/MRESOURCES-99 -->
```

Comment:
First, the link seems not working. It should be https://issues.apache.org/jira/browse/MRESOURCES-99 ?

Also, can we just use the latest maven resources plugin (3.2.0)?
Third, why we need timestamp? The existing code is using `project.info.buildtime`?

Reply:
Updated the URL.

I tested using Maven Resources plugin (3.2.0). While I was able to expand maven.build.timestamp in the properties file. I was not able to change its formatting to include milliseconds.